vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+
+$$
+\hat{x}_{t,d} = \hat{\theta_d}(t^*) \times \sum_{t=t^*-d}^{t^*} x_{t,d-1}
+$$
+
+*Note:* this amounts to effectively taking the sum across the columns in the bottom left (`block_bottom_left` in the code) of the matrix up until column $d$ and multiplying by the factor to estimate the value of $x_{t,d}$ in each row.
+This process is repeated iteratively, from bottom left to top right, to impute the bottom right triangle of the reporting matrix for each time $t$ and delay $d$ when $t+d>t^*$. The combination of the imputed and observed reporting matrix is then used to compute the delay distribution $\hat{\pi}(d)$ as described above.
+The factor, $\theta_d$ can only be computed for delays greater than 0, which effectively means that a row of the reporting triangle without any observations, can not be imputed as the process works iteratively. Therefore, the reporting matrix must have at least an entry (though it can be 0) for $x_{t*,d=0}$ to compute a nowcast for $t^*$. We will describe the method for estimating nowcasts for zero-valued counts below.
+The method described above uses the incomplete reporting triangle to iteratively compute a point nowcast. We can also use the delay distribution $\hat{\pi(d)}$ directly to compute a point nowcast, which can be useful if, for example, the delay distribution is estimated from different strata or as part of a separate estimate.
+
+### Point nowcast from delay distribution $\pi(d)$
+
+We start by computing the expected total number of eventual observed cases $\hat{x}_t$, for each reference time $t$, by summing over all delays $d$ that have already been observed (up until $t^*-t$) and dividing by the cumulative sum of the delay distribution, $\pi(d)$ up until $d = t^*-t$.
+
+$$
+\hat{x}_t= \frac{\sum_{d=1}^{d=t^*-t} x_{t,d}}{\sum_{d=1}^{d=t^*-t} \\pi(d)}
+$$
+
+Then we can compute $\hat{x}_{t,d}$ directly using the $d$th element of $\pi(d)$
+
+$$
+\hat{x}_{t,d} = \pi(d) \times \hat{x}_t
+$$
+
+Where the number of reports at timepoint $t$ with delay $d$ is the product of the the expected total reports, $x_t$ and the proportion expected at that particular delay $d$, $\pi(d)$.
+
+### Estimate of uncertainty in the nowcast
+
+To extend these point nowcasts to probabilistic nowcasts, we can use the past nowcast errors. We will describe two methods for doing this, both of which generate retrospective reporting triangles to replicate would would have been available as of time $t^*=s^*$.
+
+#### Uncertainty estimate via iteratively re-estimating the delay distribution and computing retrospective nowcasts
+
+The first method uses the retrospective incomplete reporting triangle to re-estimate a delay distribution using the $N$ preceding rows of the reporting triangle before $s^*$, and using it to recompute a retrospective nowcast, for $M$ realizations of the retrospective reporting triangle (so $M$ different $s^*$ values).
```

Comment:
as discussed face to face we are going to generalise this so it can use any set of historic or retrospective point nowcasts and then provide wrapper tools for more general cases

Reply:
removed